### PR TITLE
Support the webpack rsync plugin

### PR DESCRIPTION
**Pull request by jelly**
These changes allow build artifacts to be synced to a test virtual machine using `RSYNC=c npm run watch`

### CODE REVIEW

File: src/components/storagePools/storagePoolOverviewTab.jsx [hunk 1]
```diff
@@ -23,8 +23,6 @@ import { DescriptionList, DescriptionListGroup, DescriptionListTerm, Description
 import { storagePoolId } from '../../helpers.js';
 import cockpit from 'cockpit';
 
-import 'form-layout.scss';
```
Comment (martinpitt) [inline src/components/storagePools/storagePoolOverviewTab.jsx:26]: This is a bit unclear and buggy in the current code -- a lot of files (such as this one) include the CSS, but the only place where it is actually being used is src/components/vm/overview/bootOrder.jsx . That one would need to be ported first.

File: Makefile [hunk 2]
```diff
@@ -200,7 +200,7 @@ test/reference: test/common
 # checkout Cockpit's PF/React/build library; again this has no API stability guarantee, so check out a stable tag
```
Comment (martinpitt) [inline Makefile:200]: The "Update cockpit's lib to 260" commit mentions "podman"; perhaps say "our CSS"?

File: webpack.config.js [hunk 3]
```diff
@@ -7,12 +8,16 @@ const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 const CompressionPlugin = require("compression-webpack-plugin");
 const ESLintPlugin = require('eslint-webpack-plugin');
 const CockpitPoPlugin = require("./src/lib/cockpit-po-plugin");
+const CockpitRsyncPlugin = require("./src/lib/cockpit-rsync-plugin");
```
Comment (martinpitt) [inline webpack.config.js:11]: This should get the corresponding README changes.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Sad that this is blocked on form-layout, though :cry:

**Comment by martinpitt** [comment]
The e2e failures about `npm install` are about the internal npm mirror going down; investigating. If there is no quick fix, I'll just drop it.

**Comment by jelly** [comment]
> Thanks! Sad that this is blocked on form-layout, though cry

I'll take a look at porting, I dropped the include and visually it looked ok to me so I wasn't 100% sure if the include was required.

**Comment by martinpitt** [comment]
> I dropped the include and visually it looked ok to me

If there's no visual difference, then perhaps it actually is obsolete; then I suppose we should just rename `ct-form` class to something else, along with the rule in src/components/vm/overview/bootOrder.css ?

**Comment by jelly** [comment]
> > I dropped the include and visually it looked ok to me
> 
> If there's no visual difference, then perhaps it actually is obsolete; then I suppose we should just rename `ct-form` class to something else, along with the rule in src/components/vm/overview/bootOrder.css ?

Ok, this is now blocked on https://github.com/cockpit-project/cockpit-machines/pull/191/files or at least porting it to use a DataList and DescriptionList. Leaving the draggable for later.

File: src/components/storagePools/storagePoolOverviewTab.jsx (hunk 1 above)
Comment (jelly) [reply src/components/storagePools/storagePoolOverviewTab.jsx:26]: bootOrder.jsx is now ported in 1d680c5833cbd7bcf57d26cd67bff3cc3a976a26

File: Makefile (hunk 2 above)
Comment (jelly) [reply Makefile:200]: Copy pasta-error, adjusted.

File: webpack.config.js (hunk 3 above)
Comment (jelly) [reply webpack.config.js:11]: Done.

**Comment by jelly** [comment]
Everything should be ok now, let's see if the pixel tests agree.

**Review by martinpitt — APPROVED** [review]
!! Thanks! :100:

**Pull request merged**